roles/kubernetes-apps/network_plugin/calico/tasks/main.yml

```diff
@@ -7,5 +7,13 @@
     resource: "{{item.item.type}}"
     filename: "{{kube_config_dir}}/{{item.item.file}}"
     state: "latest"
-  with_items: "{{ calico_node_manifests.results }}"
-  when: inventory_hostname == groups['kube-master'][0] and not item|skipped
+  with_items:
+    - "{{ calico_node_manifests.results }}"
```

Comment:
this looks weird. it would pass the array as a single item

Reply:
it does what's written, actually it's just formatting change from previous file.